test/js-api/exception/constructor.tentative.any.js

```diff
@@ -0,0 +1,62 @@
+// META: global=window,dedicatedworker,jsshell
+// META: script=/wasm/jsapi/assertions.js
+
+test(() => {
+  assert_function_name(
+    WebAssembly.Exception,
+    "Exception",
+    "WebAssembly.Exception"
+  );
+}, "name");
+
+test(() => {
+  assert_function_length(WebAssembly.Exception, 1, "WebAssembly.Exception");
+}, "length");
+
+test(() => {
+  assert_throws_js(TypeError, () => new WebAssembly.Exception());
+}, "No arguments");
+
+test(() => {
+  const argument = new WebAssembly.Tag({ parameters: [] });
+  assert_throws_js(TypeError, () => WebAssembly.Exception(argument));
+}, "Calling");
```

Comment:
Why does this throw?

Reply:
The issue here is the absence of the `new` keyword, which leads to `NewTarget` being **undefined**, which causes an exception in step 1.2 of the ["create an interface object"](https://heycam.github.io/webidl/#create-an-interface-object) algorithm.